CHANGELOG.md

```diff
@@ -6,6 +6,9 @@
 ### Changed
 - Remove support for Ruby < 2.3
 
+### Fixed
+- Reading configuration from environment for boolean values
```

Comment:
Could we improve this description? The functionality change here is we aren't testing for existence anymore and have specific boolean-looking strings that we look for, correct?

Reply:
@iloveitaly updated!